apps/condo/package.json

```diff
@@ -89,6 +90,7 @@
     "raw-loader": "^4.0.2",
     "rc-queue-anim": "^1.8.5",
     "react": "^16.13.1",
+    "react-amplitude-hooks": "^0.10.6",
```

Comment:
this library with just a 25 stars and ~200 - 300 loc 
do we really need it?

Reply:
@pahaz Official amplitude react library no longer maintained and uses outdated react tools that don't match the style of the code in this project. This is just simple wrapper over it, that provides hoc-styled usage